docs/cloud-guidance/azure/ukho-azure-tagging.md

```diff
@@ -0,0 +1,28 @@
+# UKHO Azure Tagging Strategy v0.1 DRAFT
+
+## How to use this document
+
+This document outlines which tags should be applied when creating resources in the cloud. Use the table below to ensure you are conforming to the UKHO tagging strategy.
```

Comment:
```suggestion
This document outlines which tags must be applied when creating resources in the Azure. 
```
Changing `should` to `must` to be compliant with [RFC2119](https://www.ietf.org/rfc/rfc2119.txt)
Changed `cloud` to `Azure` because it should be Azure.
Removed last sentence because it was just obvious words.